src/proto/grpc/testing/control.proto

```diff
@@ -37,17 +37,32 @@ package grpc.testing;
 enum ClientType {
   SYNC_CLIENT = 0;
   ASYNC_CLIENT = 1;
+  CLIENT_VARIANT_1 = 2;
```

Comment:
I'm curious here; what do you think would be cases where a lower-bound would be interesting? On servers, for example, I always feel that it's nice to have both single-channel and multi-channel numbers to get a better idea of the impact of contention.